README.md

```diff
@@ -226,6 +226,10 @@ MEILISEARCH_REBUILD_SECRET=secret
 
 # Enable job scheduling (for cron based tasks)
 JOB_SCHEDULING=true
+
+# OPEN AI
+# OPENAI_GPT_VERSION=<DEFAULT_GPT_VERSION> # valid values are gpt-4 or gpt-3.5-turbo
+OPENAI_API_KEY=<token>
```

Comment:
We don't need org?

Reply:
The ORG one is optional, it does not look like it changes a lot, but it helps with billing I think ...